scripts/build-server

```diff
@@ -22,6 +22,6 @@ CGO_ENABLED=0 go build -tags k8s \
 
 if  [ -n "$CATTLE_KDM_BRANCH" ]; then
     curl -sLf https://releases.rancher.com/kontainer-driver-metadata/${CATTLE_KDM_BRANCH}/data.json > bin/data.json
-else
-    echo '{}' > bin/data.json
+elif [ ! -e bin/data.json ] && [ -e ../kontainer-driver-metadata/data/data.json ]; then
```

Comment:
Why did we change the build server script logic? I'm okay with this change. I'm just trying to understand its context within the PR.

Reply:
It's not related to this change.  It just prevents local builds from working correctly.  This got broken at some point during the 2.6 development cycle.